factory.go

```diff
@@ -55,6 +55,9 @@ type Parser interface {
 func NewFilecoinParser(lib *rosettaFilecoinLib.RosettaConstructionFilecoin, cacheSource common.DataSource, logger *zap.Logger, opts ...Option) (*FilecoinParser, error) {
 	defaultOpts := FilecoinParserOptions{
 		metrics: metrics.NewNoopMetricsClient(),
+		config: parser.Config{
+			FeesAsColumn: false,
+		},
```

Comment:
This config should be received from fil indexing, as we will use this component there!

Reply:
thats the default one. Desired config can be passed with the option
https://github.com/Zondax/fil-parser/blob/57128c7e1e0c8c3f360d283a002ed39d3d0d0002/options.go#L28-L33